### DISCUSSION TITLE
add Hyper-v to windows images

### DISCUSSION THREAD

**Discussion by Stromweld** [actions/virtual-environments#9285] [Installed software]
having windows hyper-v preinstalled will help with packer building vagrant boxes for hyper-v platform to be used by users with windows workstations.

**Comment by Stromweld** [comment]
I figured out it's preinstalled already but not enabled. If this could be enabled now that general runners are running on larger machines with nested virtualization available.

**Comment by mikhailkoliada** [comment]
@Stromweld hello! As you rightly mentioned it _is_ enabled :) and is available on larger hosted runners but not on the standard ones yet, so if there is an urgent need of you or business to have it I'd recommend using the larger runners for the time being
